arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -3496,16 +3609,7 @@
           "description": "Name of the connector."
         },
         "connectorType": {
-          "type": "string",
-          "enum": [
-            "CRM",
-            "AzureBlob",
-            "Salesforce"
-          ],
-          "x-ms-enum": {
-            "name": "ConnectorTypes",
-            "modelAsString": false
```

Comment:
breaking change: `"modelAsString": false` changed to `"modelAsString": true` (we encourage the latter as it prevents breaking changes in the future, so that's cool - but it's a breaking change right now)